colexecdisk: flake in TestExternalSortMemoryAccounting

Seen on unrelated PR [here](https://teamcity.cockroachdb.com/viewLog.html?buildId=6211383&buildTypeId=Cockroach_BazelEssentialCi).
```
Failed
=== RUN   TestExternalSortMemoryAccounting
    test_log_scope.go:162: test logs captured to: /artifacts/tmp/_tmp/198a389e436ad34251dab945edaab4d7/logTestExternalSortMemoryAccounting1295469402
    test_log_scope.go:80: use -show-logs to present logs inline
    external_sort_test.go:168: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1495/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/colexec/colexecdisk/colexecdisk_test_/colexecdisk_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/colexec/colexecdisk/external_sort_test.go:168
        	Error:      	"23154944" is not greater than or equal to "23603200"
        	Test:       	TestExternalSortMemoryAccounting
        	Messages:   	maximum memory bound not satisfied: actual 23603200, expected max 23154944
    panic.go:482: -- test log scope end --
--- FAIL: TestExternalSortMemoryAccounting (0.28s)
```

Jira issue: CRDB-18884

--- a/pkg/sql/colexec/colexecdisk/external_sort_test.go
+++ b/pkg/sql/colexec/colexecdisk/external_sort_test.go
@@ -162,7 +162,7 @@ func TestExternalSortMemoryAccounting(t *testing.T) {
 	}
 	// We cannot guarantee a fixed value, so we use an allowed range.
 	expMin := memoryLimit
-	expMax := int64(float64(memoryLimit) * 1.6)
+	expMax := int64(float64(memoryLimit) * 1.8)
 	require.GreaterOrEqualf(t, totalMaxMemUsage, expMin, "minimum memory bound not satisfied: "+
 		"actual %d, expected min %d", totalMaxMemUsage, expMin)
 	require.GreaterOrEqualf(t, expMax, totalMaxMemUsage, "maximum memory bound not satisfied: "+

